run_unittests.py

```diff
@@ -4128,6 +4128,29 @@ def test_configure(self):
         self.init(testdir)
         self._run(self.mconf_command + [self.builddir])
 
+    def test_summary(self):
+        testdir = os.path.join(self.unit_test_dir, '74 summary')
+        out = self.init(testdir)
+        self.assertRegex(out, r'''
```

Comment:
this dedented stuff sucks because it makes folding not work in vim and vscode, can we do:
```python
def foo():
    self.assertRegex(out, textwrap.dedent(r'''\
        ...
        ''')
```

instead?

Reply:
Nicer like that indeed.